packages/static-wado-webserver/lib/index.mjs

```diff
@@ -1,8 +1,12 @@
 import { handleHomeRelative } from "@ohif/static-wado-util";
 import express from "express";
 import logger from "morgan";
-import ConfigPoint from "config-point";
+import { importPlugin as cpImportPlugin } from "config-point";
 import dicomWebServerConfig from "./dicomWebServerConfig.mjs";
+import "regenerator-runtime";
+
+const importPlugin = (name) =>
```

Comment:
importPlugin is a factory for cpImportPlugin, so maybe worth adding it to a common place and avoid repeating ourselves

Reply:
Unfortunately that doesn't work because import is a context sensitive function so we need to provide the local version of import.